source/Irrlicht/CGLTFMeshFileLoader.cpp

```diff
@@ -311,13 +311,16 @@ void CGLTFMeshFileLoader::MeshExtractor::copyTCoords(
  * Required: NO
  * @returns: core::vector2df
 */
-float CGLTFMeshFileLoader::MeshExtractor::getScale() const
+core::vector3df CGLTFMeshFileLoader::MeshExtractor::getScale() const
 {
-	if (m_model.nodes[0].scale.size() > 0) {
-		return static_cast<float>(m_model.nodes[0].scale[0]);
+	core::vector3df buffer{1,1,1};
+	//! Fixme: What if there is more than one node?

```

Comment:
Please remove the comment to avoid confusion, since we do not plan to support more than one node.

Reply:
Fixed in 036c72cc5424d7868dfdcff08ccc6a830c7de1a9